crates/madara/src/commands/orchestrator.rs

```diff
@@ -62,3 +85,83 @@ fn build_image(shell: &Shell) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+fn populate_orchestrator_env(prover_config: &ProverRunnerConfig) -> anyhow::Result<()> {
+    let env_template = format!(
+        "{}/{}",
+        ORCHESTRATOR_REPO_PATH, ORCHESTRATOR_ENV_TEMPLATE_FILE
+    );
+    let env_output = format!("{}/{}", ORCHESTRATOR_REPO_PATH, ORCHESTRATOR_ENV_FILE);
+
+    // Read the template file
+    let template = fs::read_to_string(env_template).expect("Failed to read .env.template");
+
+    // Set up MiniJinja
+    let mut env = Environment::new();
+    env.add_template("env_template", &template)
+        .expect("Failed to add template");
+
+    let data = context! {MADARA_ORCHESTRATOR_ATLANTIC_API_KEY => prover_config.url};
```

Comment:
If the user just pressed enter, without adding an API_KEY, this will be "ATLANTIC_API_KEY",right?

Reply:
Exactly!